src/psyclone/tests/test_files/dynamo0p3/testkern_chi_2.F90

```diff
@@ -6,18 +6,30 @@
 !-------------------------------------------------------------------------------
 ! Author R. Ford STFC Daresbury Lab
 
-module testkern_chi_mod
-  type, extends(kernel_type) :: testkern_chi_type
+module testkern_chi_2
```

Comment:
You can add yourself to the author list now :-)

Reply:
Done.